### DISCUSSION TITLE
A way to disable transliteration for post slugs?

### DISCUSSION THREAD

**Discussion by quewon** [GetPublii/Publii#1910] [Help]
Hello! I'm a developer from Korea, I recently started using Publii and it's been super smooth and simple to use, even having everything in Korean. The only thing is, when it comes to Korean websites I don't think transliterating the post URLs looks very good. If having Asian characters in it creates problems, I would even prefer it to be a randomly generated string of numbers.

Is there a way to change how post slugs are generated in any way? Or is it currently possible to create a plugin that does that?

**Comment by dziudek** [comment]
@quewon - I will just add option to disable transliteration in the next version. In fact I didn't know that it can be an issue :)

**Comment by dziudek** [comment]
@quewon - I have made a deeper analysis and unfortunately it is not easy to implement simple option for that. Slug function is used in many places and in many contexts (not only for URLs). So global disable option is not the solution. Also a way how slug generation works makes it hard to easily handle new option. 

I must to postpone this task due need of further analysis. I will try to implement it in v.0.47 or v.0.48 if I will find a way to do it without huge amount of refactor

**Reply by gpsblues** [reply]
Hi, I hope I'm not saying something incorrect. But what about a general option that sets the slug equal to the ID? The slug would still exist, but it would have a simple numeric value.

**Comment by quewon** [comment]
Dang, I would have thought it would be simple since users can manually change the slug... But thank you for keeping this in consideration :)
